crates/router/src/connector/adyen/transformers.rs

```diff
@@ -311,6 +335,74 @@ pub enum PaymentType {
     Klarna,
     Affirm,
     Afterpaytouch,
+    Eps,
+    Ideal,
+    Giropay,
+    #[serde(rename = "directEbanking")]
+    Sofort,
+}
+
+pub struct AdyenTestBankNames<'a>(&'a str);
```

Comment:
We could use a better name, maybe remove `Test`?

Reply:
Adyen has different banknames for test and production hence the `Test` in the name.